.github/workflows/main.yml

```diff
@@ -11,17 +11,23 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
     steps:
-      - uses: actions/checkout@v4
+      - name: Check out Git repository
+        uses: actions/checkout@v4
 
-      - name: Install uv
-        uses: astral-sh/setup-uv@v4
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Install packages
+        run: |  
+          python -m pip install --upgrade pip  
+          pip install pytest uv
+
       - name: Run tests
-        run: uv run --with python-dateutil --with six test.py
+        run: uv run --with python-dateutil --with six -m pytest
```

Comment:
I'm confused by uv run -- but I"d think we'd want to test the isntelled package -- i.e., some version of:

```
pip install .
pytest --pyargs bson
```
Though maybe uv run does the same thing??

Reply:
Merged :)